lib/confirmation.rb

```diff
@@ -0,0 +1,27 @@
+require 'twilio-ruby'
+
+ACCOUNT_SID = ENV['TWILIO_SID']
+AUTH_TOKEN = ENV['TWILIO_TOKEN']
+RESTAURANT_PHONE = ENV['TWILIO_PHONE']
+CUSTOMER_PHONE = ENV['MY_PHONE']
+
+class Confirmation
+  def initialize(order_time = Time.now)
+    @order_time = order_time
+    @delivery_time = order_time + 3600
+    @delivery_time_hour = @delivery_time.hour
+    @delivery_time_minute = @delivery_time.min
```

Comment:
You should try to keep as little in state as possible. Considering the `delivery_time` variables all are derived from `@order_time` it's better to just store `@order_time` and calculate the others when you need them, rather than calculating at the top and storing in an instance variable.